src/ml_gym/persistency/io.py

```diff
@@ -115,13 +298,13 @@ class GridSearchAPIClientType(Enum):
     GRID_SEARCH_RESTFUL_API_CLIENT = GridSearchRestfulAPIClient
 
 
-@dataclass
+@ dataclass
```

Comment:
there should be no space here. see also other methods on top and below

Reply:
based on the documentation it works even with the space. 
Removed the space.